### PR TITLE
[READY] Update FAQ

### PR DESCRIPTION
**Pull request by sn3p**
**This can be merged once we deploy https://github.com/DefactoSoftware/LearningSpaces/pull/1673**

- [x] en
- [x] nl
- [x] de

screen shot 2016-06-06 at 16 03 25

screen shot 2016-06-06 at 16 03 45

screen shot 2016-06-06 at 15 53 09

@DefactoSoftware/learningspaces your opinions on these texts please ❓

### CODE REVIEW

**Comment by RonaldDijkstra** [comment]
Second question, I would remove the first sentence and start with: 

> This propably means... 

Because that is the answer to the question.

**Comment by sn3p** [comment]
### EN:
screen shot 2016-06-09 at 12 43 35

### NL:
screen shot 2016-06-09 at 12 42 11

File: locales/en.yml [hunk 1]
```diff
@@ -368,3 +368,21 @@ en:
...
+              After you enable JavaScript you'll need to refresh the page for the changes to take effect.
+              If this doesn't solve the problem please contact your system administrator.
+
+              Your web browser currently has JavaScript <noscript class="js-disabled">disabled</noscript>
+              <script type="text/javascript">document.write('<span class="js-enabled">enabled</span>');</script>.
+            id: error-javascript-disabled
+          - q: I get an error saying my browser isn't supported. What should I do?
+            a: |
+              This error means your browser is outdated and therefore incompatible with LearningSpaces.
+              We recommend updating your browser or the latest version to try a different brower.
```
Comment (ismay) [inline locales/en.yml]: We recommend updating your browser ~~or~~ **to** the latest version ~~to~~ **or** try a different brower.

Comment (sn3p) [inline locales/en.yml]: Haha damn @ismay, nice catch 💃

**Pull request merged**